Dockerfile

```
@@ -0,0 +1,25 @@
+FROM python:3.11-slim as base
+
+# install poetry 
+RUN pip install poetry 
+ 
+WORKDIR /opt/todoapp
+COPY . /opt/todoapp
+
+# Configure for development
+
+FROM base as development
+#install dependencies 
+RUN poetry install
+EXPOSE 5000
+ENTRYPOINT ["poetry", "run", "flask", "run", "--host=0.0.0.0"]
+
+# Configure for production
+
+FROM base as production
+#install dependencies 
+RUN poetry install
+EXPOSE 5001
```

Note that this EXPOSE is inaccurate, Gunicorn is actually running on 8000!
```suggestion
EXPOSE 8000
```